apps/atomy-q/API/app/Adapters/QuotationIntelligence/ProviderQuoteContentProcessor.php

```
@@ -0,0 +1,173 @@
+                    ? (string) $line['unit']
+                    : (string) ($rfqLineItem->uom ?? 'EA'),
+                'currency' => is_string($line['currency'] ?? null) && trim((string) $line['currency']) !== ''
+                    ? (string) $line['currency']
+                    : (string) ($rfqLineItem->currency ?? $result['currency'] ?? 'USD'),
+                'terms' => is_string($line['terms'] ?? null) ? (string) $line['terms'] : ($result['payment_terms'] ?? ''),
+                'bbox' => ['x' => 0, 'y' => 0, 'w' => 0, 'h' => 0],
+            ];
+        }
+
+        return $lines;
+    }
```

PHPMD (2.15.0)

[warning] The method lines() has a Cyclomatic Complexity of 16. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)